### DISCUSSION TITLE
Stop bots from impersonating organizations

### DISCUSSION THREAD

**Discussion by quinton-ashley** [community/community#102118] [Other Feature Feedback, Questions, & Ideas]
GitHub's current policy on name squatting is so flawed.

This organization was created by an account that's only purpose was to steal the p5play organization name from me and impersonate the p5play dev team. They could be doing this as part of a phishing scam.
https://github.com/p5play

Please give the organization name to me!

**Comment by samus-aran** [comment]
Hi @quinton-ashley 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
